Sources/TripKit/model/API/RouteAPIModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  RouteAPIModel.swift
+//  TripKit
+//
+//  Created by Adrian Schönig on 27/10/2022.
+//  Copyright © 2022 SkedGo Pty Ltd. All rights reserved.
+//
+
+import Foundation
+
+extension TKAPI {
```

Comment:
no particular comments on this one, but would like to know the reason behind having this as an extension of `TKAPI`. Is it to group API related models under `TKAPI`?

Reply:
Yes, all the `Codable` structs that mirror the basic data models as used by our API are grouped under `TKAPI`